src/store/auth/authActions.ts

```diff
@@ -1,9 +1,24 @@
+import { ASYNC_STORAGE_KEYS } from '@baca/constants'
 import { deleteToken } from '@baca/services'
 import { store } from '@baca/store/store'
+import AsyncStorage from '@react-native-async-storage/async-storage'
 
 import { isSignedInAtom } from './authState'
 
 export async function signOut() {
-  await deleteToken()
+  // set user logged out
   store.set(isSignedInAtom, false)
+
+  // remove auth token from device
```

Comment:
Maybe this logic could be placed in `NotificationService` as `deletePushToken` 🤔 

Should we check if remove from BE went successful and only then remove PushToken from Async and set `ASYNC_STORAGE_KEYS.WAS_PUSH_TOKEN_SEND` to false 🤔

Reply:
Good point, I will move this logic to NotificationService and leave just call to this function here.

> Should we check if remove from BE went successful and only then remove PushToken from Async and set ASYNC_STORAGE_KEYS.WAS_PUSH_TOKEN_SEND to false 🤔

It's not that easy:
1. We will not remove push token from async storage, because it will be used later when user will log in to other account (or the same)
2. We should always set `ASYNC_STORAGE_KEYS.WAS_PUSH_TOKEN_SEND` to false because when user will be logged out he should have possibility to send token to backend again.